tools/stonith_admin.c

```diff
@@ -53,6 +54,12 @@ static struct crm_option long_options[] = {
     {   "broadcast", no_argument, NULL, 'b',
         "Broadcast wherever appropriate."
     },
+    {   "format", required_argument, NULL, 0,
+        "Specify the format for output, one of: text, xml"
+    },
+    {   "dest", required_argument, NULL, 0,
+        "Specify the destination for formatted output, \"-\" for stdout or a filename"
+    },
```

Comment:
I'm thinking --output-as and --output-to, to distinguish from e.g. an input format

Reply:
Yeah, those are way better names.